pkg/asset/agent/manifests/util_test.go

```diff
@@ -210,6 +210,13 @@ func getProxyValidOptionalInstallConfig() *agent.OptionalInstallConfig {
 	return validIC
 }
 
+// getAdditionalTrustBundleValidOptionalInstallConfig returns a valid optional install config with AdditonalTrustBundle
```

Comment:
linter warning :roll_eyes: 
```suggestion
// getAdditionalTrustBundleValidOptionalInstallConfig returns a valid optional install config with AdditonalTrustBundle.
```

Reply:
Fixed